.circleci/dockerfiles/wasm-stable

```diff
@@ -0,0 +1,7 @@
+FROM emscripten/emsdk:3.1.29
```

Comment:
In case you didtn' know, Dockerfiles do support an [`ARG`](https://docs.docker.com/engine/reference/builder/#arg) directive for supplying build-time arguments via [`docker build --build-arg`](https://docs.docker.com/engine/reference/commandline/build/#-set-build-time-variables---build-arg). It's up to you whether you want to have separate Dockerfiles anyway, since I could see some utility in that, especially if you needed to change other stuff in the Dockerfile (maybe the list of apt packages?), but I don't think it's a huge deal either way because this Dockerfile is small and we already have a plethora of Dockerfiles for the different Linux distros.

Reply:
Done!